### PR TITLE
added new relic and artillery for stress testing

### PR DESCRIPTION
**Pull request by mjones61692-zz**
@dweinzimmer, please take a look at these changes I made for stress testing with New Relic and Artillery / JMeter.

### CODE REVIEW

File: spec/idMaker.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const generateId = (userContext, events, done) => {
+  const id = Math.ceil(Math.random() * 10000000);
+  userContext.vars.id = id;
+  return done();
+};
+
+module.exports = {
```
Comment (dweinzimmer) [inline spec/idMaker.js:7]: Not a requirement by any means, but if you want to you can just set the entire function to `module.exports = (userContext, events, done) => ...` if you want to to make this even more concise and visually apparent that the only thing the module does is provide the helper function. But it's very readable as is.

File: spec/stressTest.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    headers:
+      x-my-service-auth: "987401838271002188298567"
+scenarios:
+  - name: "visit page"
+    flow:
+      - loop:
+        - function: "generateId"
+        - get:
+            url: "/description?id={{ id }}"
+        count: 10
```
Comment (dweinzimmer) [inline spec/stressTest.yml]: Github is suggesting a new line return at the end of this file.

**Comment by dweinzimmer** [comment]
Hi @mjones61692, I made a couple comments in line, but like you said this looks very straightforward. Feel free to merge!

**Comment by mjones61692-zz** [comment]
@dweinzimmer, thank you! I added a line at the the end of that file.

**Pull request merged**